nzedb/Logger.php

```diff
@@ -62,7 +62,7 @@ class Logger
 	 * @var bool
 	 * @access private
 	 */
-	private $outputCLI = true;
+	private $outputCLI;
```

Comment:
This is wrong, please change it as it changes the semantics of the assignment (It will default to null/unset).